### PR TITLE
02 11 2020

### PR DESCRIPTION
**Pull request by BlakeTurner**

### CODE REVIEW

File: how-were-organized.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+#### Flex
+
+The Flex track is the KTLO/Support function of the Product Engineering Group (See [How we invest our time](how-we-invest-our-time.md)). The Flex track works closely with BAS (Bench App Support) and BizTech to handle the issues, outages, and stakeholder requests that come our way during a cycle and its cooldown. In effect, the Flex track makes the Focus track _possible_ by absorbing the myriad distractions the team needs to deal with during a cycle. The number of engineers on the Flex track scales based on total number of clients, a proxy for business complexity.
+
+### Design
+
+The Design team is asked to concretely point toward a future that doesn’t exist yet. They work closely with the Product team to deeply understand problems, and with the Product Engineers to deeply understand potential solutions, and use that understanding to define in real terms how the next thing we build will work and what it will look like. They work directly alongside engineerings in the Focus track to bring these solutions into reality.
```
Comment (matt-lam-bench) [inline how-were-organized.md:66]: typo: `engineerings` -> `engineering`

File: how-were-organized.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
+# How we're organized
+
+[![Overview of how the Technology Teams are organized](images/groups-and-functions.png)](images/groups-and-functions.png)
+
+Creating great technology is a Bench-wide effort. In our day-to-day work, we mostly interface with [7 teams](#groups-and-teams) who report into three departments: `Technology`, `Product`, and `Operations`. The teams are organized into two working groups: `Business Technology (BizTech)` and `Product Development`.
+
+To prevent siloing, we have a Technical Leadership group who ensure that the efforts of our teams are complimentary, and that our systems are in good health. They work closely with the Backend and Frontend guilds to create alignment around how we build our technology.
```
Comment (matt-lam-bench) [inline how-were-organized.md:7]: typo: `complimentary` -> `complementary`

File: technology-principles.md [hunk 3]
```diff
@@ -12,17 +12,17 @@ _[Learn how to use these principles](culture-and-principles.md)_
...
+- Growth happens when an individual continuously asks themselves: is this the best I can do?
+- To grow, we must step into conversations that feel scary and uncomfortable.
 - To foster the growth of our colleagues—regardless of their job title—we must give them deeply honest feedback, and expect them to do the same for us.
 - Feedback is most effective when those receiving it feel supported by those giving it.
 - Successes and failures are important because of what we learn from them.
 
 ## Honour Time
-- By intentionally limiting the amount of time we spend on our problems, we are adding a constraint that asks us to be continuously creative while solving them.
+- Limited time == more creativity
+- Overtime is cheating!
```
Comment (matt-lam-bench) [inline technology-principles.md:23]: I understand the desire to make these more concise, but does it make the idea (and how these two assertions are related) less approachable for someone reading this the first time?

Comment (BlakeTurner) [reply technology-principles.md:23]: @matt-lam-bench Does this apply to both assertions or just the `overtime is cheating` one?

Comment (matt-lam-bench) [reply technology-principles.md:23]: both

Comment (matt-lam-bench) [reply technology-principles.md:23]: My suggestions:
`Limited time == more creativity` -> `We set deadlines to push ourselves to be creative in solving problems.`
`Overtime is cheating!` -> `Overtime is cheating: Working extra hours short-circuits the time constraint that we've set for ourselves keeps us from being creative.`

File: technology-principles.md [hunk 4]
```diff
@@ -12,17 +12,17 @@ _[Learn how to use these principles](culture-and-principles.md)_
...
+- Growth happens when an individual continuously asks themselves: is this the best I can do?
+- To grow, we must step into conversations that feel scary and uncomfortable.
 - To foster the growth of our colleagues—regardless of their job title—we must give them deeply honest feedback, and expect them to do the same for us.
 - Feedback is most effective when those receiving it feel supported by those giving it.
 - Successes and failures are important because of what we learn from them.
 
 ## Honour Time
-- By intentionally limiting the amount of time we spend on our problems, we are adding a constraint that asks us to be continuously creative while solving them.
+- We set deadlines to push ourselves to be creative in solving problems.
+- Overtime is cheating: Working extra hours short-circuits the time constraint that we've set for ourselves keeps us from being creative.
```
Comment (matt-lam-bench) [inline technology-principles.md:23]: Oops, I have a typo in my own suggestion!

`... set for ourselves keeps us from being creative.` -> `... set for ourselves, keeping us from being creative.`

I'll go ahead and push a correction myself

**Review by matt-lam-bench — APPROVED** [review]

**Pull request merged**